.github/actions/release-notes-aggregator/action.yml

```diff
@@ -1,16 +1,16 @@
 name: Release Notes Aggregator
 description: 'Creates a release notes by aggregating external repositories release notes.'
 inputs:
-  externalRepo:
+  external-repo:
     description: 'The name of the external repositories to aggregate release notes from'
     required: true
-  generateRNfromVersion:
+  generate-rn-from-version:
```

Comment:
from-version and to-version are enough to understand what they are doing in this context